.env.tmp

```diff
@@ -89,3 +89,4 @@ EDGE_DEVICE_ID=""
 CREDENTIALS=""
 SERVICE_PRINCIPAL=""
 UPDATE_DOTENV=""
+VERSION="0.0.1"
```

Comment:
This entry is for a specific test only, so I prefer not adding it here but only adding it to that test.

Reply:
updated code